src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerNative.tsx

```diff
@@ -91,22 +91,21 @@ function Controls({
   const seconds = String(timeRemaining % 60).padStart(2, '0')
 
   useEffect(() => {
-    const interval = setInterval(() => {
-      // duration gets reset to 0 on loop
-      if (player.duration) setDuration(Math.floor(player.duration))
-      setCurrentTime(Math.floor(player.currentTime))
-
-      // how often should we update the time?
-      // 1000 gets out of sync with the video time
-    }, 250)
-
-    // eslint-disable-next-line @typescript-eslint/no-shadow
-    const sub = player.addListener('volumeChange', ({isMuted}) => {
-      setIsMuted(isMuted)
-    })
-
+    // const interval = setInterval(() => {
+    //   // duration gets reset to 0 on loop
+    //   if (player.duration) setDuration(Math.floor(player.duration))
+    //   setCurrentTime(Math.floor(player.currentTime))
+    //
+    //   // how often should we update the time?
+    //   // 1000 gets out of sync with the video time
+    // }, 250)
```

Comment:
Any idea what we need to do to uncomment this? :/

Reply:
Yea, I have a plan to fix this today.